coopetition/scoring_program/score.py

```diff
@@ -33,8 +33,17 @@
     # prediction[config['X_cols']] = reference[config['X_cols']]
 
     score = 0
-    for col in config['Y_cols']:
-        score = max(score, ks_2samp(reference[col], prediction[col])[0])
+    kmax = 50
```

Comment:
Could you please rename `kmax` to something more meaningful? Maybe, `number_of_slices` or `n_slices`?
Also, you should read its value from `config`.